scripts/Upgrade-AzureIAM-DataCollector.ps1

```diff
@@ -0,0 +1,213 @@
+<#
+.SYNOPSIS
+    Script to upgrade Cloudneeti IAM data collector automation account
```

Comment:
Script to upgrade Cloudneeti IAM data collector's automation account

Reply:
done